learnMoreScreens/YoureNotAlone.js

```diff
@@ -48,12 +48,12 @@ class YoureNotAlone extends Component {
                         )}
                 />
                 <Text style={styles.subTitle}>
-                    <Text style={{ fontWeight: 'bold' }}>{translate('youAreNotAlone.content.videoCard.boldText')}</Text> 
+                    <Text style={{ fontFamily: 'avenir-heavy' }}>{translate('youAreNotAlone.content.videoCard.boldText')}</Text> 

```

Comment:
There was no space between bolded caption and regular caption. Updated language JSON file and pushed fix.